components/calendar/src/calendar.rs

```diff
@@ -26,14 +26,17 @@ use core::fmt;
 /// </div>
 pub trait Calendar: crate::cal::scaffold::UnstableSealed {
     /// The internal type used to represent dates
+    #[cfg_attr(not(feature = "unstable"), doc(hidden))]
```

Comment:
thought: ditto on stability here

Reply:
I am not currently convinced that the types and methods on `Date` should be marked unstable.

Reply:
please comment on the APIs you're referring to